src/components/uikit/button/index.js

```diff
@@ -101,6 +101,7 @@ export default compose(
     feedbackDuration : number.isRequired, // milliseconds
     feedbackLabel    : string.isRequired,
     confirmation     : string,
+    inactive         : bool.isRequired,
```

Comment:
Nitpick: this prop influences style / behaviour, move it a few lines up near `disabled` or wherever you think it's best.